docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+Later in 2014, [GitHub announced](https://web.archive.org/web/20140506163807/http://blog.atom.io/2014/05/06/atom-is-now-open-source.html) that Atom would become fully open-sourced under the MIT license and invited the community to actively contribute to its development; it is under this license that Pulsar is still developed.
+
+In 2015, [Atom 1.0](https://github.com/atom/atom/releases/tag/v1.0.0) was released and came with a [fantastic retro-futurisitic launch video](https://www.youtube.com/watch?v=Y7aEiVwBAdk) to accompany it.
+
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=Y7aEiVwBAdk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Microsoft acquires GitHub
+
+<img src="./assets/microsoft-github.png" height="200" />
+
+In June of 2018, Microsoft announced that they were [acquiring GitHub](https://blogs.microsoft.com/blog/2018/06/04/microsoft-github-empowering-developers/). This immediately raised concerns from the Atom user base because Microsoft had released their own Electron-based text editor back in 2015, Visual Studio Code. The concerns were that Atom would shut down development of Atom in favor of their own product as the two products were competing in very similar areas. To help alleviate the Atom community's worries, Microsoft's newly instated Github CEO, Nat Friedman, said in a [Reddit AMA](https://www.reddit.com/r/AMA/comments/8pc8mf/im_nat_friedman_future_ceo_of_github_ama/e0a2b2e/) that Atom was safe and development would continue on both products.
+
+### Tree-sitter lands in Atom
+
+<img src="./assets/tree-sitter-lands.png" height="200" />
+
+You may have read our [series of blog posts](https://pulsar-edit.dev/tag/tree-sitter/) from [@savetheclocktower] and [@maurício szabo] that detail the huge amount of work that has been going on to upgrade our Tree-sitter implementation. This all goes back to 2018 when [Max Brunsfeld announced](https://github.blog/2018-10-31-atoms-new-parsing-system/) that Atom 1.32 would be shipping with a brand new parsing system that promised to be better at understanding your code and making it easier to write - Tree-sitter. Like how Atom Shell was created for Atom and was then widely used elsewhere, the same can be said of Tree-sitter, which now sits at the heart of a number of other projects: Helix, Zed, and even Github's own [Linguist](https://github.com/github-linguist/linguist) library.
```

Comment:
```suggestion
You may have read our [series of blog posts](https://pulsar-edit.dev/tag/tree-sitter/) from [@savetheclocktower] and [@maurícioszabo] that detail the huge amount of work that has been going on to upgrade our Tree-sitter implementation.

This all goes back to 2018, when [Max Brunsfeld announced](https://github.blog/2018-10-31-atoms-new-parsing-system/) that Atom 1.32 would be shipping with a brand-new parsing system that promised to be better at understanding your code and making it easier to write: [Tree-sitter](https://tree-sitter.github.io/tree-sitter/).

Just as Atom Shell was created for Atom and was then widely used elsewhere, Tree-sitter now sits at the heart of a number of other projects: [Helix](https://github.com/helix-editor/helix), [Zed](https://zed.dev/), [Neovim](https://neovim.io/), and even Github's own [Linguist](https://github.com/github-linguist/linguist) library.
```

Reply:
Using Maurício's tag like that goes back a long way in the blogs, I don't 100% remember why it started but I just have a standard list of reference links that just get copied over between blogs. Without the refence link being updated too, this change would break it.

Reply:
That’s fine. I never did that in my articles, so I assumed it was a typo.

Reply:
Just a quirk really, and being lazy as I'd have to go and edit the refernce link. I'll pick from this suggestion and push a new commit.